app/dashboard/templates/shared/onepager_auth.html

```diff
@@ -105,5 +107,5 @@
   }
 </style>
 
```

Comment:
is this by design?

Reply:
I made a code clean of the tips, needed to be able to add the notifications in that page